projects/Sylk/services/sylkserver/module/default.nix

```diff
@@ -114,37 +98,42 @@ in
     # TODO: dynamic user?
     # there were some issues with using a dynamic user (tied to uid and gid)
     # that may warrant patching the software
-    users.groups.${cfg.group} = { };
-    users.users.${cfg.user} = {
+    users.groups.${service} = { };
+    users.users.${service} = {
       description = "SylkServer service user";
       isSystemUser = true;
-      group = cfg.group;
+      group = service;
+    };
+
+    services.${service} = {
+      settings.config.Server.trace_dir = "/var/log/${service}";
+      settings.config.Server.spool_dir = "/var/lib/${service}";
+      settings.conference.Conference.file_transfer_dir = "/var/lib/${service}/file_transfer";
+      settings.conference.Conference.screensharing_images_dir = "/var/lib/${service}/screensharing_images";
```

Comment:
I don't agree with not making these directories configurable through these options. Automatically binding these to the service directories seems like a better UX to me than demanding that users do this themselves. As a user, if options are available to me, I expect them to take effect when I change them.

Reply:
Yeah that's fair. I'll drop the commit.